view/theme/frio/theme.php

```diff
@@ -28,16 +28,23 @@ function frio_init(&$a) {
 
 	// if the device is a mobile device set js is_mobile
 	// variable so the js scripts can use this information
-	if($a->is_mobile || $a->is_tablet) {
+	if ($a->is_mobile || $a->is_tablet) {
 		$a->page["htmlhead"] .= <<< EOT
 			<script>
 				var is_mobile = 1;
 			</script>
 EOT;
 			}
 
-	if ($style == "")
+	if ($style == "") {
 		$style = get_config('frio', 'style');
+	}
+
+	// Disable the richtext editor for frio because frio don't support
+	// the richtext editor
+	if (local_user() && feature_enabled(local_user(), 'richtext')) {
+		set_pconfig(local_user(), 'feature', 'richtext', 0);
```

Comment:
I feel this is counter-intuitive for users. If they had the richtext editor enabled and switch back from frio to another theme that supports the richtext editor, they shouldn't have to reactivate it in their settings.